src/main/resources/config/maven-idea-codestyle-2022.xml

```diff
@@ -123,7 +131,7 @@ under the License.
   <codeStyleSettings language="XML">
     <indentOptions>
       <option name="INDENT_SIZE" value="2" />
-      <option name="CONTINUATION_INDENT_SIZE" value="4" />
+      <option name="CONTINUATION_INDENT_SIZE" value="2" />
```

Comment:
Is. that the value used by the formatter ?

Reply:
simply I changed my local config to format in the same way as spotles does 
this is result of my export - so probably is used 😄

Reply:
any other comments?

Reply:
ok, as it is for developer as help only I will merge. 
In case of any feedback we will simply update it.